.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: "v0.11.8"
+    rev: "v0.11.9"
```

Comment:
i think this is in collision with https://github.com/cytomining/pycytominer/pull/529